clinica/iotools/bids_utils.py

```diff
@@ -485,25 +493,23 @@ def write_scans_tsv(bids_dir, bids_ids, scans_dict):
     import os
     from glob import glob
     from os import path
-
     import pandas as pd
 
     for bids_id in bids_ids:
-        # Create the file
         sessions_paths = glob(path.join(bids_dir, bids_id, "ses-*"))
         for session_path in sessions_paths:
-            scans_df = pd.DataFrame()
             session_name = session_path.split(os.sep)[-1]
-            tsv_name = bids_id + "_" + session_name + "_scans.tsv"
-
+            scans_df = pd.DataFrame()
+            # Create the file
+            tsv_name = "_".join([bids_id, session_name, "scans.tsv"])
```

Comment:
Similar comment. You just rewrote the same line in a more idiomatic fashion, but it adds nothing of value to this PR.

Reply:
Not like it's a bad thing, but again the lighter the diff the better.